app.js

```diff
@@ -39,7 +39,7 @@ const scopes = ['identify', 'guilds'];
 passport.use(new DiscordStrategy({
     clientID: process.env.CLIENT_ID,
     clientSecret: process.env.CLIENT_SECRET,
-    callbackURL: "https://dev-resources.herokuapp.com/user/auth/discord/callback",
+    callbackURL: "http://localhost:3001/user/auth/discord/callback",
```

Comment:
This could do with being reverted so it doesnt break the build on production. 
Although this should probably be converted to the env variable

Reply:
Beat me to it :joy:

Reply:
LOL, yeah I guess env would do, last time I worked with the route, I almost made this mistake, this it  got me. XD